### PR TITLE
Pods Beaver Themer 1.1

### PR DESCRIPTION
**Pull request by quasel**

### CODE REVIEW

File: classes/class-pods-beaver-page-data.php [hunk 1]
```diff
@@ -159,13 +159,13 @@ public static function get_pod( $settings = array() ) {
 
 			if ( ! $pod || ! $pod->valid() ) {
 				$pod = false;
-			}
+			} elseif ( $item_id && $pod && ! $pod->exists() ) {
+				$pod = false;
+			} else {
+				self::$pods[ $pod_name ] = $pod;
+			};
```
Comment (sc0ttkclark) [inline classes/class-pods-beaver-page-data.php]: ; here not needed.

File: pods-beaver-themer.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Pods Beaver Themer Add-On
  * Plugin URI: http://pods.io/
  * Description: Integration with Beaver Builder Themer (https://www.wpbeaverbuilder.com). Provides a UI for mapping Field Connections with Pods
- * Version: 1.0
+ * Version: 1.1-a-1
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: 1.1 now?

File: pods-beaver-themer.php [hunk 3]
```diff
@@ -124,7 +124,7 @@ function pods_beaver_fake_loop_false() {
  *
  * @param null|object $settings
  *
- * @since 1.0
+ * @since 1.1
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: Should be 1.0 still, right?

File: pods-beaver-themer.php [hunk 4]
```diff
@@ -218,38 +222,50 @@ function pods_beaver_loop_settings_before_form( $settings ) {
  *
  * @return object
  *
- * @since 1.0
+ * @since 1.1
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: Should be 1.0 still, right?

File: pods-beaver-themer.php [hunk 5]
```diff
@@ -218,38 +222,50 @@ function pods_beaver_loop_settings_before_form( $settings ) {
...
+			'name'   => trim( $settings->field ),
+		);
+	} /*elseif ( 'pods_advanced' === $settings->use_pods && ! empty( $settings->pods_where ) ) {
 		$find_params = array(
 			'where' => trim( $settings->pods_where ),
 			'limit' => - 1,
 		);
+	}*/
+
+	$find_params = apply_filters('pods_beaver_loop_settings_find_params', $find_params, $settings, $pod);
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: Hook docs for this? https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

File: pods-beaver-themer.php [hunk 6]
```diff
@@ -218,38 +222,50 @@ function pods_beaver_loop_settings_before_form( $settings ) {
...
+	$field_params = apply_filters('pods_beaver_loop_settings_field_params', $field_params, $settings, $pod);
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: Hook docs for this? https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

File: pods-beaver-themer.php [hunk 7]
```diff
@@ -301,3 +320,41 @@ function pods_beaver_empty_query() {
...
+ */
+function pods_beaver_render_settings_field_order_by($field, $name, $settings) {
+
+	if ( 'order_by' == $name ) {
+		$field['options']['post__in'] = __('Preserve Relationship (pick) Order', 'pods-beaver-builder-themer-add-on');
+	}
+
+	return $field;
+}
+
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: Remove extra line break

File: pods-beaver-themer.php [hunk 8]
```diff
@@ -301,3 +320,41 @@ function pods_beaver_empty_query() {
...
+/**
+ * Add Option to order_by settings field
+ *
+ * @return array $field
+ *
+ * @since 1.1
+ */
+function pods_beaver_render_settings_field_order_by($field, $name, $settings) {
+
+	if ( 'order_by' == $name ) {
```
Comment (sc0ttkclark) [inline pods-beaver-themer.php]: ===

File: readme.txt [hunk 9]
```diff
@@ -4,15 +4,15 @@ Donate link: https://pods.io/friends-of-pods/
 Tags: pods, beaver builder, beaver themer,
 Requires at least: 4.4
 Tested up to: 4.8
-Stable tag: 1.0
+Stable tag: 1.1-a-1
```
Comment (sc0ttkclark) [inline readme.txt]: 1.1

File: readme.txt [hunk 10]
```diff
@@ -56,16 +56,25 @@ We are growing a community of Beaver Themer users who are starting to hang out o
 
 If you've uncovered a Bug or have a Feature Request, we kindly request you to [Create an issue on GitHub](https://github.com/pods-framework/pods-beaver-builder-themer-add-on/issues/new). Please be very specific about what steps you did to create the issue you're having and include any screenshots or other configuration parameters to help us recreate or isolate the issue.
 
+= HowTo Test upcoming Versions =
```
Comment (sc0ttkclark) [inline readme.txt]: `= How to test upcoming versions =`

File: readme.txt [hunk 11]
```diff
@@ -56,16 +56,25 @@ We are growing a community of Beaver Themer users who are starting to hang out o
...
-2. Example dropwdown for different field types.
+2. Example dropdown for different field types.
 3. Support for all types of Field Connections like Color.
-4. Example dropwdown for color.
+4. Example dropdown for color.
 5. Example for Templates or Magic Tags.
 
 == Changelog ==
 
+= 1.1 - June 9th, 2017 =
```
Comment (sc0ttkclark) [inline readme.txt]: June 14th, 2017

File: pods-beaver-themer.php (hunk 3 above)
Comment (quasel) [inline pods-beaver-themer.php]: well we had it in there just no activated - but valid point

File: pods-beaver-themer.php (hunk 4 above)
Comment (quasel) [inline pods-beaver-themer.php]: well we had it in there just no activated - but valid point

**Pull request merged**